mttl/models/expert_model.py

```diff
@@ -587,6 +592,17 @@ def as_expert(self):
             "This method is not implemented for MultiExpertModel."
         )
 
+    def add_empty_experts(self):
```

Comment:
this func should be in MoE model I think

Reply:
yes true